### PR TITLE
EDSC-2805: Hides features when portal authentication is disabled

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: static/src/js/components/GranuleResults/GranuleResultsTableHeaderCell.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import GranuleResultsDataLinksButton from './GranuleResultsDataLinksButton'
 import Button from '../Button/Button'
 import MoreActionsDropdown from '../MoreActionsDropdown/MoreActionsDropdown'
 import MoreActionsDropdownItem from '../MoreActionsDropdown/MoreActionsDropdownItem'
+import { authenticationEnabled } from '../../util/portals'
```
Comment (trevorlang) [inline static/src/js/components/GranuleResults/GranuleResultsTableHeaderCell.js:12]: I think this should be above the components

File: static/src/js/components/GranuleResults/GranuleResultsItem.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ import MoreActionsDropdown from '../MoreActionsDropdown/MoreActionsDropdown'
 import MoreActionsDropdownItem from '../MoreActionsDropdown/MoreActionsDropdownItem'
 
 import './GranuleResultsItem.scss'
+import { authenticationEnabled } from '../../util/portals'
```
Comment (trevorlang) [inline static/src/js/components/GranuleResults/GranuleResultsItem.js:16]: This should sit above the components

File: static/src/js/components/GranuleResults/GranuleResultsActions.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ import Button from '../Button/Button'
 import Skeleton from '../Skeleton/Skeleton'
 
 import './GranuleResultsActions.scss'
+import { authenticationEnabled } from '../../util/portals'
```
Comment (trevorlang) [inline static/src/js/components/GranuleResults/GranuleResultsActions.js:16]: This should sit above the components

File: static/src/js/components/CollectionResults/CollectionResultsTableHeaderCell.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 import Button from '../Button/Button'
+import { authenticationEnabled } from '../../util/portals'
```
Comment (trevorlang) [inline static/src/js/components/CollectionResults/CollectionResultsTableHeaderCell.js:5]: This should sit above the components

File: static/src/js/components/CollectionResults/CollectionResultsItem.js [hunk 5]
```diff
@@ -11,6 +11,7 @@ import Button from '../Button/Button'
 import SplitBadge from '../SplitBadge/SplitBadge'
 
 import './CollectionResultsItem.scss'
+import { authenticationEnabled } from '../../util/portals'
```
Comment (trevorlang) [inline static/src/js/components/CollectionResults/CollectionResultsItem.js:14]: This should sit above the components

File: static/src/js/components/CollectionResults/CollectionResultsHeader.js [hunk 6]
```diff
@@ -12,6 +12,7 @@ import Skeleton from '../Skeleton/Skeleton'
 import { collectionResultsTotal } from './skeleton'
 
 import './CollectionResultsHeader.scss'
+import { authenticationEnabled } from '../../util/portals'
```
Comment (trevorlang) [inline static/src/js/components/CollectionResults/CollectionResultsHeader.js:15]: This should sit above the components

File: static/src/js/components/SecondaryToolbar/SecondaryToolbar.js [hunk 7]
```diff
@@ -10,10 +10,11 @@ import Button from '../Button/Button'
 import PortalLinkContainer from '../../containers/PortalLinkContainer/PortalLinkContainer'
 import { getEnvironmentConfig } from '../../../../../sharedUtils/config'
 import { cmrEnv } from '../../../../../sharedUtils/cmrEnv'
-
-import './SecondaryToolbar.scss'
 import { portalPath } from '../../../../../sharedUtils/portalPath'
 import { stringify } from '../../util/url/url'
+import { authenticationEnabled } from '../../util/portals'
```
Comment (trevorlang) [inline static/src/js/components/SecondaryToolbar/SecondaryToolbar.js:15]: This should sit above the components

File: static/src/js/components/CollectionResults/CollectionResultsHeader.js [hunk 8]
```diff
@@ -161,17 +162,20 @@ const CollectionResultsHeader = ({
...
-              <strong className="collection-results-header__tip-label">Tip:</strong>
-              Add
-              <i className="collection-results-header__tip-icon fa fa-plus" />
-              collections to your project to compare and download their data.
-            </span>
-          </div>
-        </div>
-
+        {
+          authenticationEnabled(portal) && (
```
Comment (abbottry) [inline static/src/js/components/CollectionResults/CollectionResultsHeader.js:166]: Can we move this to a component that looks at the portal value from the store and wraps the conditional components, passing them as children? The component can be connected.

**Pull request merged**